docs/scenarios_authoring/Fragments.md

```diff
@@ -9,14 +9,34 @@ Fragments are a way to share processing logic - if many scenarios require identi
+
+![fragment input_advanced_long](img/fragment_input_advanced_long.png)
+
+While for parameters of type `String` or `Boolean`, on top of that it also allows choosing an input mode from `Any`, `Any with suggestions`, `Fixed list` and, if applicable, setting the items of the list.
+
+![fragment input_advanced_string](img/fragment_input_advanced_string.png)
+
+| Field name               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|--------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Required                 | If on, the parameter's value has to be provided (the field cannot be left empty when using the fragment)      .                                                                                                                                                                                                                                                                                                                                                                                           |
+| Validation               | If on, only literal values (e.g. `123`) and operations on them (e.g. `123 * 321`) are allowed as the parameter's value. This is because Nussknacker has to be able to evaluate it at deployment time.<br/>If on, allows definition of `Validation expression` and `Validation error message`.                                                                                                                                                                                                             |
+| Validation expression    | SpEL expression that evaluates to a `true`/`false` result, that depends only on the parameter's value (referred to as `#value`) and literal values, e.g. `#value < 100`. <br/> When using the fragment, values that don't satisfy the expression will cause a validation error.                                                                                                                                                                                                                           |
```

Comment:
`, that depends only on the parameter's value` -> `. Only parameter's value (referred to as ..) and literal values are allowed in validation expression, e.g`.